### PR TITLE
gsutil: add page

### PR DESCRIPTION
**Pull request by dnafication**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/gsutil.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+> You can use gsutil to do a wide range of bucket and object management tasks.
+> More information: <https://cloud.google.com/storage/docs/gsutil>.
+
+- List all buckets in a project you are logged into:
+
+`gsutil ls`
+
+- List the objects in a bucket:
+
+`gsutil ls -r gs://{{BUCKET_NAME}}/{{PREFIX}}**`
```
Comment (EmilySeville7cfg) [inline pages/common/gsutil.md:13]:
```suggestion
`gsutil ls -r gs://{{bucket_name}}/{{prefix}}**`
```

File: pages/common/gsutil.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Download an object from a bucket:
+
+`gsutil cp gs://{{BUCKET_NAME}}/{{OBJECT_NAME}} {{SAVE_TO_LOCATION}}`
```
Comment (EmilySeville7cfg) [inline pages/common/gsutil.md:17]:
```suggestion
`gsutil cp gs://{{bucket_name}}/{{object_name}} {{path/to/save_location}}`
```

File: pages/common/gsutil.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Upload an object to a bucket:
+
+`gsutil cp {{OBJECT_LOCATION}} gs://{{DESTINATION_BUCKET_NAME}}/`
```
Comment (EmilySeville7cfg) [inline pages/common/gsutil.md:21]:
```suggestion
`gsutil cp {{object_location}} gs://{{destination_bucket_name}}/`
```

File: pages/common/gsutil.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Rename or move objects in a bucket:
+
+`gsutil mv gs://{{BUCKET_NAME}}/{{OLD_OBJECT_NAME}} gs://{{BUCKET_NAME}}/{{NEW_OBJECT_NAME}}`
```
Comment (EmilySeville7cfg) [inline pages/common/gsutil.md:25]:
```suggestion
`gsutil mv gs://{{bucket_name}}/{{old_object_name}} gs://{{bucket_name}}/{{new_object_name}}`
```

File: pages/common/gsutil.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+- Create a new bucket in the project you are logged into:
+
+`gsutil mb gs://{{BUCKET_NAME}}`
```
Comment (EmilySeville7cfg) [inline pages/common/gsutil.md:29]:
```suggestion
`gsutil mb gs://{{bucket_name}}`
```

File: pages/common/gsutil.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- Delete a bucket and remove all the objects in it:
+
+`gsutil rm -r gs://{{BUCKET_NAME}}`
```
Comment (EmilySeville7cfg) [inline pages/common/gsutil.md:33]:
```suggestion
`gsutil rm -r gs://{{bucket_name}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/gsutil.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/gsutil.md:13]: We use lower case for placeholders. But there are some exceptions: placeholders with SQL code for instance.

File: pages/common/gsutil.md (hunk 4 above)
Comment (dnafication) [reply pages/common/gsutil.md:25]: Thanks for changing these to the right format 😄

**Comment by dnafication** [comment]
@EmilySeville7cfg thanks for reviewing. I've accepted all recommendations.

**Review by blueskyson — APPROVED** [review]

File: pages/common/gsutil.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# gsutil
+
+> The gsutil CLI is a Python application that lets you access Cloud Storage from the command line.
```
Comment (pixelcmtd) [inline pages/common/gsutil.md:3]:
```suggestion
> The gsutil CLI lets you access Google Cloud Storage from the command line.
```

File: pages/common/gsutil.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+> You can use gsutil to do a wide range of bucket and object management tasks.
+> More information: <https://cloud.google.com/storage/docs/gsutil>.
+
+- List all buckets in a project you are logged into:
+
+`gsutil ls`
+
+- List the objects in a bucket:
+
+`gsutil ls -r gs://{{bucket_name}}/{{prefix}}**`
```
Comment (pixelcmtd) [inline pages/common/gsutil.md:13]:
```suggestion
`gsutil ls -r 'gs://{{bucket_name}}/{{prefix}}**'`
```

**Review by pixelcmtd — APPROVED** [review]

**Comment by dnafication** [comment]
I don't see any other changes that needs to be addressed. @EmilySeville7cfg can you confirm?

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**